actions/version-release/action.yml

```diff
@@ -0,0 +1,27 @@
+name: "Version Release"
```

Comment:
What will be the trigger to call this action? Not sure about the use case.

Reply:
Here is an example of how I plan to use this action

https://github.com/SkynetLabs/ansible-playbooks/pull/286/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R47

This is part of enable continuous release for our repos